cob_cartesian_controller/scripts/lwa4d/lwa4d_circle.py

```diff
@@ -0,0 +1,136 @@
+#! /usr/bin/env python
```

Comment:
can you move them to the `schunk_lwa4d` package? There, create a new folder `scripts`!